### PR TITLE
Renamed all @this variables.

### PR DESCRIPTION
**Pull request by michelvosje**
That was an easy fix for issue: #383

### CODE REVIEW

File: generators/server/templates/dotnetcore/src/Project/Configuration/MvcStartup.cs.ejs [hunk 1]
```diff
@@ -22,56 +22,56 @@ using Newtonsoft.Json.Serialization;
...
             {
                 configuration.RootPath = "ClientApp/dist";
             });
 
-            return @this;
+            return services;
         }
 
-        public static IApplicationBuilder UseApplicationWeb(this IApplicationBuilder @this, IHostEnvironment env)
+        public static IApplicationBuilder UseApplicationWeb(this IApplicationBuilder appBuilder, IHostEnvironment env)
```
Comment (nicolas63) [inline generators/server/templates/dotnetcore/src/Project/Configuration/MvcStartup.cs.ejs:51]: In other extension we use app can you do the same to maintain consistency ?

**Review by nicolas63 — CHANGES_REQUESTED** [review]

**Review by nicolas63 — APPROVED** [review]
Nice ! thank you for your first contribution here 🎉

**Comment by michelvosje** [comment]
The build issue does not seem to be caused by my changes?

**Comment by nicolas63** [comment]
Yes i have re-run the jobs

**Pull request merged**